src/tracker.cu

```diff
@@ -216,7 +211,7 @@ namespace Tracker {
      */
     void vortPos(const int *marker, std::vector<std::shared_ptr<Vtx::Vortex> > &vLocation, int xDim, const double2 *wfc){
         int i,j;
-        unsigned int counter=0;
+        // unsigned int counter=0;
```

Comment:
Again, if these are not used, can they be removed?

Reply:
I noticed a few cases in which variables seemed to exist to observe something: here, I think counter was used to manually verify if something was called the right number of times.
I commented these out so that the compiler stopped complaining about unused variables, but thought it would be best to leave them in case of debugging.
If there are any pieces of commented code that you don't think this does/should apply to, make a comment here and I'll remove them.